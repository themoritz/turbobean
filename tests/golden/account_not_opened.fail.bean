2020-01-02 * "Buy"
  Assets:Cash  -10 EUR
  Expenses:Home

