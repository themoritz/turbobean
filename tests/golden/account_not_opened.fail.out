account_not_opened.fail.bean: [Error] Account is not open or has been closed. Open it with an open entry

    2 |   Assets:Cash  -10 EUR
          ^^^^^^^^^^^

account_not_opened.fail.bean: [Error] Account is not open or has been closed. Open it with an open entry

    3 |   Expenses:Home
          ^^^^^^^^^^^^^

